atspi-common/src/events/object.rs

```diff
@@ -1138,25 +1211,59 @@ impl From<StateChangedEvent> for EventBodyOwned {
 }
 
 impl_from_user_facing_event_for_interface_event_enum!(
-	ChildrenChangedEvent,
+	ChildrenInsertedEvent,
+	ObjectEvents,
+	ObjectEvents::ChildrenInserted
+);
+impl_from_user_facing_type_for_event_enum!(ChildrenInsertedEvent, Event::Object);
+impl_try_from_event_for_user_facing_type!(
+	ChildrenInsertedEvent,
+	ObjectEvents::ChildrenInserted,
+	Event::Object
+);
+event_test_cases!(ChildrenInsertedEvent);
+impl_to_dbus_message!(ChildrenInsertedEvent);
+impl_from_dbus_message!(ChildrenInsertedEvent);
+impl_event_properties!(ChildrenInsertedEvent);
+
+impl_from_user_facing_event_for_interface_event_enum!(
+	ChildrenDeletedEvent,
 	ObjectEvents,
-	ObjectEvents::ChildrenChanged
+	ObjectEvents::ChildrenDeleted
 );
-impl_from_user_facing_type_for_event_enum!(ChildrenChangedEvent, Event::Object);
+impl_from_user_facing_type_for_event_enum!(ChildrenDeletedEvent, Event::Object);
 impl_try_from_event_for_user_facing_type!(
-	ChildrenChangedEvent,
-	ObjectEvents::ChildrenChanged,
+	ChildrenDeletedEvent,
+	ObjectEvents::ChildrenDeleted,
 	Event::Object
 );
-event_test_cases!(ChildrenChangedEvent);
-impl_to_dbus_message!(ChildrenChangedEvent);
-impl_from_dbus_message!(ChildrenChangedEvent);
-impl_event_properties!(ChildrenChangedEvent);
-impl From<ChildrenChangedEvent> for EventBodyOwned {
-	fn from(event: ChildrenChangedEvent) -> Self {
+event_test_cases!(ChildrenDeletedEvent);
+impl_to_dbus_message!(ChildrenDeletedEvent);
+impl_from_dbus_message!(ChildrenDeletedEvent);
+impl_event_properties!(ChildrenDeletedEvent);
+
+impl From<ChildrenInsertedEvent> for EventBodyOwned {
+	fn from(event: ChildrenInsertedEvent) -> Self {
+		EventBodyOwned {
+			properties: std::collections::HashMap::new(),
+			kind: "insert".to_string(),
```

Comment:
Should this not be "add"?

Reply:
With busmonitoring I saw ChildrenChanged: "add/system", "remove/system" and "add", "remove"
So this should probably be "add" or "add/system"

Reply:
Yes.